static/templates/buddy_list_tooltip_content.hbs

```diff
@@ -1,5 +1,9 @@
 <div class="buddy_list_tooltip_content">
-    <span class="tooltip_inner_content">{{first_line}}</span>
+    <span class="tooltip_inner_content">{{first_line}}
+        {{#if show_you}}
+        <span class="my_user_status">(you)</span>
```

Comment:
The `you` needs to be tagged for translation, like all user facing strings :).

https://github.com/zulip/zulip/blob/master/docs/translating/internationalization.md

Reply:
Ohh Yes!